src/main/java/uk/gov/justice/laa/crime/cfecrime/utils/RequestHandler.java

```diff
@@ -2,44 +2,98 @@
 
 import lombok.experimental.UtilityClass;
 import lombok.extern.slf4j.Slf4j;
-import uk.gov.justice.laa.crime.cfecrime.api.Result.Outcome;
+import uk.gov.justice.laa.crime.cfecrime.Exceptions.UndefinedOutcomeException;
 import uk.gov.justice.laa.crime.cfecrime.api.*;
+import uk.gov.justice.laa.crime.cfecrime.api.stateless.StatelessApiRequest;
+import uk.gov.justice.laa.crime.cfecrime.api.stateless.StatelessApiResponse;
+import uk.gov.justice.laa.crime.cfecrime.cma.stubs.LocalCmaService;
+import uk.gov.justice.laa.crime.cfecrime.enums.Outcome;
+import uk.gov.justice.laa.crime.meansassessment.staticdata.enums.CaseType;
+import uk.gov.justice.laa.crime.meansassessment.staticdata.enums.FullAssessmentResult;
+import uk.gov.justice.laa.crime.meansassessment.staticdata.enums.InitAssessmentResult;
+import uk.gov.justice.laa.crime.meansassessment.staticdata.enums.MagCourtOutcome;
 
 @Slf4j
 @UtilityClass
 public class RequestHandler {
+    private LocalCmaService cmaService = new LocalCmaService();
+    private StatelessApiResponse statelessApiResponse = null;
+    private StatelessApiRequest cmaRequest = new StatelessApiRequest();
+    private CaseType caseType =  null;
+    private MagCourtOutcome magCourtOutcome = null;
 
-    public static CfeCrimeResponse handleRequest(CfeCrimeRequest request){
+    public static CfeCrimeResponse handleRequest(CfeCrimeRequest cfeCrimeRequest ) throws UndefinedOutcomeException {
 
         Boolean under18 = null;
         Boolean passported = null;
-        Outcome oc = null;
+        Outcome outcome = null;
 
-        if (request.getSectionUnder18() != null) {
-            under18 = request.getSectionUnder18().getClientUnder18();
+        if (cfeCrimeRequest.getSectionUnder18() != null) {
+            under18 = cfeCrimeRequest.getSectionUnder18().getClientUnder18();
         }
-        if (request.getSectionPassportedBenefit() != null) {
-            passported = request.getSectionPassportedBenefit().getPassportedBenefit();
+        if (cfeCrimeRequest.getSectionPassportedBenefit() != null) {
+            passported = cfeCrimeRequest.getSectionPassportedBenefit().getPassportedBenefit();
         }
-        oc = getOutcomeFromAgeAndPassportedBenefit(under18,passported);
+        outcome = getOutcomeFromAgeAndPassportedBenefit(under18, passported);
+
+        CfeCrimeResponse cfeCrimeResponse = new CfeCrimeResponse();
+        if (outcome != null) {
+            cfeCrimeResponse.setOutcome(outcome);
+        }else{
+            //call cma
+            statelessApiResponse = cmaService.callCma(cmaRequest);
```

Comment:
Think this should be a local variable

Reply:
This is being used by other functions in the class.

Reply:
but it shouldn't be. We have a conscious policy to avoid side effects